provider/datagen/src/baked_exporter.rs

```diff
@@ -132,6 +132,8 @@ pub struct BakedExporter {
     impl_data: Mutex<BTreeMap<&'static str, ImplData>>,
     // List of dependencies used by baking.
     dependencies: CrateEnv,
+    // The data's fallback mode
+    fallback_mode: Mutex<FallbackMode>,
```

Comment:
It's a fixed config setting on `DatagenProvider`, as you can see below. However, the exporter needs to be made aware of it so that it generates the correct metadata, and therefore it has to be communicated through the `DataExporter` trait (exporter construction is not controlled by us).

Reply:
obsolete, now passed in `flush`